geotrek/feedback/models.py

```diff
@@ -631,16 +629,16 @@ def try_send_email(self, subject, message, report=None):
                 self.attach_email_to_report(report, message, self.user.email)
 
     def notify_report_to_solve(self, report):
-        subject = _("Geotrek - A report must be solved")
-        message = render_to_string("feedback/cloture_email.txt", {"report": report})
+        subject = str(settings.EMAIL_SUBJECT_PREFIX) + str(_("A report must be solved"))
```

Comment:
concat string by using %s, .format() or f"strings". No need to cast string with str